bin/evaluate.sh

```diff
@@ -2,6 +2,13 @@
 
 set -o errexit
 
+backtrace="--backtrace"
+if [ "${*#*$backtrace}" != "$*" ]; then
+    export RUST_BACKTRACE=1
+    set -- ${*%$backtrace*}${*#*$backtrace}
```

Comment:
Oh I forgot, looks like execute() is calling playpen.execute() which doesn't allow setting any env. vars ... and this is why I had to set RUST_BACKTRACE within `evaluate.sh` depending on the incoming args (`--backtrace`)

What do you think?